### ISSUE TITLE
Multiple partition support per Sink instance

### ISSUE THREAD

**Issue by sobychacko** [spring-cloud/spring-cloud-stream#2336]
@tajdars commented on [Wed Feb 21 2018](https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit/issues/127)

Moved from 
https://github.com/spring-cloud/spring-cloud-stream/issues/1232

I've a RabbitMQ binder based Stream Sink Service with parent set to :
<parent>
	<groupId>org.springframework.cloud</groupId>
	<artifactId>spring-cloud-build</artifactId>
	<version>1.3.7.RELEASE</version>
	<relativePath /> 
</parent>

The input Topic is partitioned,
 How can we get more than 1 Topic partition assigned to a Sink Service instance ?

Basically we want to limit the # of Sink Service instances but still get the parallelism across partitions.

Thanx,

---

@olegz commented on [Wed Feb 21 2018](https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit/issues/127#issuecomment-367328089)

@tajdars Thank you for moving it! As I mentioned we'd be definitely interested in contribution from the community so if you up for the challenge go ahead and give it a shot and submit a PR and we'll work with you to make it happen.
Thanks again!
